.gitignore

```diff
@@ -1,17 +1,50 @@
-.DS_Store
-*.lock
-examples/ssl/ssl.json
-examples/ssl/server
-examples/ssl/client
-examples/ssl/sensu_ca
-.vagrant
-\#*
-.bundle/
+*.rbc
+.config
+coverage
+InstalledFiles
+lib/bundler/man
+pkg
+rdoc
+spec/reports
+test/tmp
+test/version_tmp
+tmp
+_Store
+*~
+*#
+.#*
+\#*#
+.*.sw[a-z]
+*.un~
+*.tmp
+*.bk
+*.bkup
+
+# ruby/bundler files
+.ruby-version
+.ruby-gemset
+.rvmrc
+Gemfile.lock
+.bundle
+*.gem
+
+# YARD artifacts
+.yardoc
+_yardoc
+doc/
+.idea
+
+#chef stuff
+Berksfile.lock
```

Comment:
At the moment we're still using librarian-chef in this project. Will you please add `Berkshelf` to gitignore as well? Looks like that line was lost in this revision.